golos.social/golos.social.abi

```diff
@@ -87,23 +78,10 @@
      {"name": "block",        "type": "block"},
      {"name": "unblock",      "type": "block"},
      {"name": "updatemeta",   "type": "updatemeta"},
-     {"name": "deletemeta",   "type": "delete"},
+     {"name": "deletemeta",   "type": "deletemeta"},
   ],
   "events": [ ],
   "tables": [
-     {
```

Comment:
Applying of this ABI will fail, because table has records.
Only empty tables can be removed.